subsys/net/openthread/Kconfig

```diff
@@ -63,10 +63,9 @@ config OPENTHREAD_RADIO_WORKQUEUE_STACK_SIZE
 	default 1084 if SOC_NRF5340_CPUAPP
 	default 1024
 
-# TF-M support in OpenThread is experimental
-config BUILD_WITH_TFM
+config TFM_MINIMAL
```

Comment:
Why this is changed this way? Currently for nRF5340 NS we have CONFIG_BUILD_WITH_TFM=y and CONFIG_TFM_MINIMAL=n . If TFM stops to be experimental the whole config should be simply removed from here.

Reply:
Are you saying that you have `CONFIG_TFM_MINIMAL=n` for all samples that require it, and that I don't need to have the configuration set default here?

Setting TFM_MINIMAL=n is experimental, so this still produces a build warning about experimental configuration being enabled.

Reply:
Run on sdk-nrf/main:
_west build -b nrf5340dk_nrf5340_cpuapp_ns -p always nrf/samples/openthread/cli_
you will see that it produces build warning:
_warning: Experimental symbol BUILD_WITH_TFM is enabled._
and configuration:
CONFIG_BUILD_WITH_TFM=y
# CONFIG_TFM_MINIMAL is not set

Reply:
@lmaciejonczyk

`west build -b nrf5340dk_nrf5340_cpuapp_ns -p always samples/openthread/cli`

Produces the following build warning:
`warning: Experimental symbol TFM_EXPERIMENTAL is enabled.`

And the following configurations:
```
CONFIG_BUILD_WITH_TFM=y
# CONFIG_TFM_MINIMAL is not set
```